LICENSE.md

```diff
@@ -19,3 +19,5 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+Note that this license doesn't apply to code under the collector/compile-benchmarks directory.
```

Comment:
I would put this at the top perhaps or we can do a setup similar to rust-lang/rust with the COPYRIGHT file which lists out exceptions.

Reply:
While we're doing that I think if we can just get permission to do MIT OR Apache 2.0 that would also be good (for all directories) rather than just MIT.

Reply:
Moved it to the top.